src/tests/header/headerSelection.ts

```diff
@@ -104,10 +109,24 @@ const isLastOneOffContributionWithinLast2To13Months = (
 const getNonSupportersTest = (edition: string): HeaderTest =>
     edition === 'UK' ? nonSupportersTestUK : nonSupportersTestNonUK;
 
+const isAusMoment = (countryCode: string): boolean => countryCode === 'AU' && isAusMomentLive();
+
+// date is set to present for testing purposes - needs to be changed to july 17th when aus moment goes live
+const isAusMomentLive = () => Date.now() >= Date.parse('2021-06-19');
+
 export const selectHeaderTest = (
     targeting: HeaderTargeting,
 ): Promise<HeaderTestSelection | null> => {
     const select = (): HeaderTest => {
+        if (isAusMoment(targeting.countryCode)) {
+            if (targeting.lastOneOffContributionDate) {
+                return ausMomentOneOffContributor;
```

Comment:
We might want a cut off here? i.e just one off supporters within the last 6 months? I'd check with Victoria about what that should be